Samples/Sample.AspNetCore2.WebApi/Startup.cs

```diff
@@ -29,6 +29,10 @@ public void ConfigureServices(IServiceCollection services)
             //{"Endpoint Routing does not support 'IApplicationBuilder.UseMvc(...)'. To use 'IApplicationBuilder.UseMvc' set 'MvcOptions.EnableEndpointRouting = false' inside 'ConfigureServices(...)."}
             services.AddMvc(options => options.EnableEndpointRouting = false);
             services.AddOptions();
+            //services.Configure<IISServerOptions>(options =>
+            //{
+            //    options.AllowSynchronousIO = true;
+            //});
```

Comment:
Is this related to the other commented out code above? If so, should it be removed as well?